### DISCUSSION TITLE
[WASM] How to suppress the 'UNO is loading your application' startup message?

### DISCUSSION THREAD

**Discussion by DierkDroth** [unoplatform/uno#9636] [Q&A] [answered]
For a short period of time (could be a split second) there is a 'UNO is loading your application' message showing right on the mouse pointer if you hovered the cursor over the splashscreen while starting the app.
You e.g. could reproduce using the repo [from this case](https://github.com/unoplatform/uno/issues/9625) and does not appear to be related to a particular splash screen.

No offense to UNO intended, but for CI (Corporate Identity) reasons I needed to have that message suppressed. 

How could that be achieved?

**Comment by FrozDark** [comment, marked as answer]
https://platform.uno/docs/articles/splash-screen.html?tabs=tabid-vswin#step-5---webassembly

**Reply by DierkDroth** [reply]
@FrozDark thanks for your feedback. 

I'm aware of the logic to implement a custom splash screen. However - as I tried to point out - the issue persists regardless of the actual (custom) splash screen.

What am I missing?

**Reply by jeromelaban** [reply]
This message [comes from here](https://github.com/unoplatform/Uno.Wasm.Bootstrap/blob/22fea3d1fd5521ac0befefa8ba4b1cea77ca2dbd/src/Uno.Wasm.Bootstrap/Templates/index.html#L22), you can remove it by changing the index.html file.

**Reply by DierkDroth** [reply]
Thanks @jeromelaban 

However this brings back the problem which I had [with a custom index.html](https://github.com/unoplatform/uno/discussions/9597). Which index.html should I use to not run into that trouble again?

Note: the 'Templates/index.html' link [here ](https://platform.uno/docs/articles/external/uno.wasm.bootstrap/doc/features-additional-files.html)is still not working.

**Reply by jeromelaban** [reply]
You'll need to use the [file associated to the version](https://github.com/unoplatform/Uno.Wasm.Bootstrap/blob/release/stable/3.3/src/Uno.Wasm.Bootstrap/Templates/index.html) of the uno bootstrapper you're using.

As for the file still not working, we're always taking PRs from the community.

**Reply by DierkDroth** [reply]
Excellent! Thanks @jeromelaban. That did the trick

**Comment by DierkDroth** [comment]
Closed
